### DISCUSSION TITLE
Problem when using the library

### DISCUSSION THREAD

**Discussion by douglascugliarisenai** [focus-shift/jollyday#533] [Q&A] [answered]
I put the library in a project of mine but the error is occurring below:

Cannot instantiate datasource instance because there is no implmentation

Class ConfigurantionServiceManager

function instantiateDataSource

**Comment by derTobsch** [comment, marked as answer]
> I put the library in a project of mine but the error is occurring below:
> 
> Cannot instantiate datasource instance because there is no implmentation
> 
> Class ConfigurantionServiceManager
> 
> function instantiateDataSource

You need to use the Jackson or jaxb dependency and not the core. I think I need to update the readme. Thanks for the hint.

see https://github.com/focus-shift/jollyday?tab=readme-ov-file#xml-binding-libraries
